Simplified/Utilities/Date-Time/Date+NYPLAdditions.swift

```diff
@@ -27,4 +40,64 @@ public extension Date {
   var rfc1123String: String {
     return Date.rfc1123DateFormatter.string(from: self)
   }
+  
+  /// A date string with the choice of short or long suffix
+  /// Example: 5 years / 5 y / 6 months / 1 day
+  func timeUntilString(suffixType: NYPLDateSuffixType) -> String {
```

Comment:
I think this replaces the objc `longTimeUntilString`, correct? if so, can you just remove that objc method too? otherwise we'd have the same function duplicated in 2 languages.

Reply:
The `longTimeUntilString` and `shortTimeUntilString` methods are for objc `NSDate` class, and they are still being called by `NYPLBookDetailNormalView.m`.
The `Date` class is not available in objc from my understanding, I will leave a note to remove the `longTimeUntilString` once we get rid of the last usage of it.

Reply:
ohh yeah you're right I *always* forget that `NSDate` is not `Date` 😂 

you could do an NSDate extension that just calls the Date methods though, maybe, something like this: https://github.com/NYPL-Simplified/Simplified-iOS/blob/develop/Simplified/Logging/URLRequest%2BLogging.swift

Reply:
Fixed

Reply:
perfect, thanks!